homeassistant/components/tankerkoenig/const.py

```diff
@@ -6,4 +6,7 @@
 CONF_FUEL_TYPES = "fuel_types"
 CONF_STATIONS = "stations"
 
-FUEL_TYPES = ["e5", "e10", "diesel"]
+DEFAULT_RADIUS = 2
+DEFAULT_SCAN_INTERVAL = 30
+
+FUEL_TYPES = {"e5": "Super", "e10": "Super E10", "diesel": "Diesel"}
```

Comment:
Shouldn't the names be translatable?

Reply:
as far as I know, it is not possible to translate values (_even options_) for inputs, only their keys are translatable 🤔